Total Instances: 81

Total Correct: 2
Total Runtime Failed: 7
Total Compilation Failed: 3
Total Test Failed: 67
Total Infinite Loop: 2

Accuracy: 2.4691358024691357
Runtime Rate: 8.641975308641975
Compilation Rate: 3.7037037037037033
Test Failed Rate: 82.71604938271605
Infinite Loop Rate: 2.4691358024691357

=================================================================================================
Failed Test Files: ['codeforces_569_A.py', 'atcoder_ABC169_D.py', 'codeforces_59_A.py', 'codeforces_92_A.py', 'codeforces_622_A.py', 'atcoder_ABC178_A.py', 'atcoder_ABC143_A.py', 'codeforces_334_A.py', 'codeforces_514_A.py', 'atcoder_ABC174_C.py', 'codeforces_546_A.py', 'codeforces_110_B.py', 'atcoder_ABC155_E.py', 'codeforces_79_A.py', 'atcoder_AGC046_A.py', 'codeforces_340_A.py', 'atcoder_ARC062_B.py', 'atcoder_ABC108_B.py', 'atcoder_AGC046_B.py', 'codeforces_579_A.py', 'atcoder_ABC149_C.py', 'codeforces_32_B.py', 'atcoder_ABC158_A.py', 'atcoder_ABC042_A.py', 'atcoder_ABC139_B.py', 'codeforces_678_A.py', 'codeforces_58_B.py', 'codeforces_672_A.py', 'codeforces_86_A.py', 'atcoder_ABC125_A.py', 'codeforces_49_A.py', 'atcoder_ABC132_F.py', 'codeforces_190_A.py', 'atcoder_ABC178_B.py', 'atcoder_ABC158_B.py', 'codeforces_96_B.py', 'atcoder_ABC124_C.py', 'codeforces_55_A.py', 'atcoder_ABC142_A.py', 'atcoder_AGC025_A.py', 'atcoder_ABC168_C.py', 'atcoder_ABC043_B.py', 'atcoder_ABC127_B.py', 'codeforces_171_A.py', 'atcoder_ABC120_C.py', 'atcoder_ABC132_A.py', 'codeforces_678_B.py', 'codeforces_276_B.py', 'atcoder_ABC124_A.py', 'codeforces_651_A.py', 'atcoder_ABC153_A.py', 'atcoder_ABC149_B.py', 'codeforces_306_A.py', 'codeforces_544_B.py', 'codeforces_242_A.py', 'codeforces_459_A.py', 'atcoder_ABC136_B.py', 'atcoder_ABC151_A.py', 'atcoder_ABC070_B.py', 'codeforces_379_A.py', 'codeforces_581_A.py', 'atcoder_ABC164_A.py', 'codeforces_8_B.py', 'codeforces_369_B.py', 'atcoder_ABC114_C.py', 'atcoder_AGC002_A.py', 'codeforces_30_A.py'] 
Failed Test Details: ['Filename: codeforces_514_A.py Actual: 4304 Generated: 4221\n', 'Filename: atcoder_ABC168_C.py Actual: -45.96915777646216 Generated: 1.000076150972172\n', 'Filename: atcoder_AGC046_B.py Actual: -2 Generated: 65\n', 'Filename: codeforces_544_B.py Actual: 96 Generated: NO\n', 'Filename: atcoder_ABC142_A.py Actual: 27.935654253757406 Generated: 0.5\n', 'Filename: atcoder_ABC132_F.py Actual: 217 Generated: 147\n', 'Filename: atcoder_ABC136_B.py Actual: 90933 Generated: 90909\n', 'Filename: atcoder_ABC158_A.py Actual: Yzes Generated: Yes\n', 'Filename: atcoder_ABC153_A.py Actual: 10046 Generated: 10000\n', 'Filename: codeforces_276_B.py Actual: SeconBd Generated: Second\n', 'Filename: atcoder_ABC149_B.py Actual: -49 1000000000008 Generated: 0 1000000000000\n', 'Filename: codeforces_569_A.py Actual: -20 Generated: 5\n', 'Filename: codeforces_30_A.py Actual: 13 Generated: 2\n', 'Filename: atcoder_ABC149_C.py Actual: 9513 Generated: 9587\n', 'Filename: atcoder_ABC108_B.py Actual: 3 10 -1 7 -80 Generated: 3 10 -1 7\n', 'Filename: codeforces_96_B.py Actual: 4444477694 Generated: 7444447777\n', 'Filename: codeforces_622_A.py Actual: 344 Generated: 311.0\n', 'Filename: atcoder_ABC043_B.py Actual: 78 Generated: 0000000000\n', 'Filename: atcoder_AGC002_A.py Actual: Postive Generated: Positive\n', 'Filename: codeforces_334_A.py Actual: 1 36 3 34 5 32 \n2 35 4 33 6 31 \n7 30 9 28 11 26 \n8 29 10 27 12 25 \n13 24 15 22 17 20  Generated: 1 36 3 34 5 32 \n2 35 4 33 6 31 \n7 30 9 28 11 26 \n8 29 10 27 12 25 \n13 24 15 22 17 20 \n14 23 16 21 18 19 \n', 'Filename: codeforces_86_A.py Actual: -60 Generated: 0\n', 'Filename: codeforces_672_A.py Actual: 55 Generated: 3\n', 'Filename: codeforces_579_A.py Actual: 48 Generated: 2\n', 'Filename: atcoder_ABC178_B.py Actual: -186233282690604537 Generated: -186233282690604452\n', 'Filename: codeforces_369_B.py Actual: -27 Generated: 1\n', 'Filename: codeforces_110_B.py Actual: ar;dabcda}cdabcd}bcsaWcd5bcda=cdabcdabc`aOcdab4dabodabcdafcdabcwabcdabcdabcdabgdabud[bcdabAdaocd2bkd Generated: abcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcd\n', 'Filename: codeforces_581_A.py Actual: -51 -57 Generated: 45 27\n', 'Filename: atcoder_AGC025_A.py Actual: 34 Generated: 9\n', 'Filename: codeforces_58_B.py Actual: 733035\n244345\n48869\n1\n-36 Generated: 733035\n244345\n48869\n1\n', 'Filename: atcoder_ABC174_C.py Actual: 18 Generated: -1\n', 'Filename: atcoder_ABC132_A.py Actual: N Generated: No\n', 'Filename: atcoder_ABC155_E.py Actual: -1 Generated: 1\n', 'Filename: atcoder_ARC062_B.py Actual: 82 Generated: -1\n', 'Filename: atcoder_ABC169_D.py Actual: 9 Generated: 10\n', 'Filename: atcoder_ABC070_B.py Actual: -21 Generated: 0\n', 'Filename: atcoder_ABC124_A.py Actual: 61 Generated: 12\n', 'Filename: codeforces_49_A.py Actual: N Generated: NO\n', 'Filename: codeforces_190_A.py Actual: 96 154 -55 Generated: 96 154\n', 'Filename: atcoder_ABC164_A.py Actual: unRsafe Generated: unsafe\n', 'Filename: codeforces_79_A.py Actual: Cqiel Generated: Ciel\n', 'Filename: codeforces_379_A.py Actual: 146 Generated: 80\n', 'Filename: codeforces_55_A.py Actual: YGk Generated: YES\n', 'Filename: codeforces_651_A.py Actual: 38 Generated: 122\n', 'Filename: atcoder_ABC143_A.py Actual: -27 Generated: 25\n', 'Filename: codeforces_306_A.py Actual: 94 -94 43 12 56 -8 101 21 92 -19 68 -19 -89 -66 30 79 92 -53 -88 -65 -88 -23 -38 -62 -66 69 -30 -28 -46 -81 -79 49 -51 76 Generated: 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 3 ', 'Filename: atcoder_ABC151_A.py Actual: O Generated: d\n', 'Filename: codeforces_92_A.py Actual: -41 Generated: 9\n', 'Filename: codeforces_678_A.py Actual: 1000000087 Generated: 1000000000\n', 'Filename: atcoder_ABC139_B.py Actual: -31 Generated: 1\n', 'Filename: atcoder_ABC127_B.py Actual: 17\n0\n11\n28\n59\n51\n132\n258\n526\n929 Generated: 3\n5\n9\n17\n33\n65\n129\n257\n513\n1025\n', 'Filename: atcoder_ABC125_A.py Actual: -46 Generated: 0\n', 'Filename: atcoder_ABC124_C.py Actual: 54 Generated: 0\n', 'Filename: codeforces_459_A.py Actual: 4 4 0 Generated: 4 4 0 0\n', 'Filename: atcoder_ABC158_B.py Actual: 370614275896614948 Generated: 370614275896614890\n', "Filename: codeforces_59_A.py Actual: VS?Z]#CVVVH@y@VB=?HO-7(152g1JC~(MyQ0^oWWw]xVx&uO2N#oUVgiDQwK^M;?9TcYuTlq4SQ.cj:|+qlxpsp[PmW .k`NWT' Generated: CDOZDPBVVVHNBJVBYHEOXWFLJKRWJCAJMIFCOZWWYFKVWOGTVJCUUSIGDQFKUMEWJTDYITVEEIAYBWRHOMRWMPDIPJWIUXFNWUZ\n", 'Filename: codeforces_171_A.py Actual: 898600302 Generated: 898600336\n', 'Filename: codeforces_340_A.py Actual: 1000000044 Generated: 1000000000\n', 'Filename: codeforces_8_B.py Actual: BU Generated: BUG\n', 'Filename: atcoder_ABC042_A.py Actual: NO Generated: YES\n', 'Filename: atcoder_AGC046_A.py Actual: 276 Generated: 360\n', 'Filename: atcoder_ABC120_C.py Actual: -27 Generated: 0\n', 'Filename: atcoder_ABC178_A.py Actual: 25 Generated: 1\n', 'Filename: atcoder_ABC114_C.py Actual: 8358 Generated: 8335\n', 'Filename: codeforces_242_A.py Actual: 73\n3 2\n4j 2\n43\nUH2\nYz(\n45 4 Generated: 6\n3 2\n4 2\n4 3\n5 2\n5 3\n5 4\n', 'Filename: codeforces_546_A.py Actual: -60 Generated: 0\n', 'Filename: codeforces_678_B.py Actual: 2141 Generated: 2108\n', 'Filename: codeforces_32_B.py Actual: 54 Generated: 100\n'] 
=================================================================================================
Runtime Error Files: ['atcoder_ABC051_A.py', 'codeforces_189_A.py', 'atcoder_ABC122_D.py', 'codeforces_203_A.py', 'atcoder_ABC169_C.py', 'codeforces_669_A.py', 'atcoder_ABC170_A.py'] 
Runtime Error Details: ['Filename: atcoder_ABC051_A.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/atcoder_ABC051_A.py", line 8, in <module>\n    main()\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/atcoder_ABC051_A.py", line 4, in main\n    a, b, c = input().split(\',\')\nValueError: too many values to unpack (expected 3)\n', 'Filename: codeforces_669_A.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/codeforces_669_A.py", line 41, in <module>\n    print(codeforces_669_A(999999991))\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/codeforces_669_A.py", line 27, in codeforces_669_A\n    fr = FastReader()\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/codeforces_669_A.py", line 6, in __init__\n    self.s = sys.stdin.read().lstrip().rstrip()\nNameError: name \'sys\' is not defined\n', 'Filename: atcoder_ABC169_C.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/atcoder_ABC169_C.py", line 9, in <module>\n    b = float(sys.stdin.readline().split()[1])\nIndexError: list index out of range\n', 'Filename: codeforces_189_A.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/codeforces_189_A.py", line 19, in <module>\n    main()\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/codeforces_189_A.py", line 4, in main\n    n = int(sys.stdin.readline())\nValueError: invalid literal for int() with base 10: \'29 27 18 2\\n\'\n', 'Filename: atcoder_ABC122_D.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/atcoder_ABC122_D.py", line 35, in <module>\n    print(calc())\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/atcoder_ABC122_D.py", line 32, in calc\n    return dfs(0, "TTT", memo)\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/atcoder_ABC122_D.py", line 20, in dfs\n    if current == N:\nNameError: name \'N\' is not defined\n', 'Filename: atcoder_ABC170_A.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/atcoder_ABC170_A.py", line 9, in <module>\n    print(atcoder_ABC170_A())\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/atcoder_ABC170_A.py", line 4, in atcoder_ABC170_A\n    x = int(input())\nValueError: invalid literal for int() with base 10: \'1 2 3 4 0\'\n', 'Filename: codeforces_203_A.py Error_type: Traceback (most recent call last):\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/codeforces_203_A.py", line 33, in <module>\n    main()\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/codeforces_203_A.py", line 29, in main\n    solver.solve(inp, out)\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/codeforces_203_A.py", line 5, in solve\n    points = inp.read_int()\n  File "/home/ali/Documents/CodeMind/translation_output/misleading_test/deepseekcoder/avatar/java/python-sanitized/codeforces_203_A.py", line 23, in read_int\n    return int(self.stream.readline())\nValueError: invalid literal for int() with base 10: \'3 1 20 15 17 5\\n\'\n'] 
=================================================================================================
Compilation Error Files: ['codeforces_99_A.py', 'atcoder_AGC007_C.py', 'codeforces_373_B.py'] 
=================================================================================================
Infinite Loop Files: ['atcoder_ARC102_C.py', 'atcoder_ABC172_D.py'] 
=================================================================================================
